src/ape_test/providers.py

```diff
@@ -76,6 +76,11 @@ def revert(self, snapshot_id: SnapshotID):
     def set_timestamp(self, new_timestamp: int):
         self._tester.time_travel(new_timestamp)
 
+    def mine(self, num_blocks: int):
```

Comment:
```suggestion
    def mine(self, num_blocks: int = 1):
```

Reply:
added this, we had it in the chain.py, just forgot to add it to the providers